README.md

```diff
@@ -2164,27 +2165,29 @@ A Guide like this can’t give you personalized advice on what a reasonable offe
 that depends greatly on your skills, the marketplace of candidates, what other offers you
 have, what the company can pay, what other candidates the company has found, and the
 company’s needs.
-However, this section covers some basics of what to expect with offers, and tips on
+However, this section covers some basics of what to expect with offers, as well as tips on
 [negotiating an offer](https://hbr.org/2014/04/15-rules-for-negotiating-a-job-offer).
```

Comment:
(Not a copy edit but noting a reminder to myself to move this link later, under negotiation tips.)

Reply:
👍

Reply:
Now done.